configs/distill/rkd/rkd_neck_resnet34_resnet18_8xb32_in1k.py

```diff
@@ -0,0 +1,72 @@
+_base_ = [
+    '../../_base_/datasets/mmcls/imagenet_bs32.py',
+    '../../_base_/schedules/mmcls/imagenet_bs256.py',
+    '../../_base_/mmcls_runtime.py'
+]
+
+# model settings
+student = dict(
+    type='mmcls.ImageClassifier',
+    backbone=dict(
+        type='ResNet',
+        depth=18,
+        num_stages=4,
+        out_indices=(3, ),
+        style='pytorch'),
+    neck=dict(type='GlobalAveragePooling'),
+    head=dict(
+        type='LinearClsHead',
+        num_classes=1000,
+        in_channels=512,
+        loss=dict(type='CrossEntropyLoss', loss_weight=1.0),
+        topk=(1, 5),
+    ))
+
+# teacher settings
+teacher = dict(
+    type='mmcls.ImageClassifier',
```

Comment:
```suggestion
    type='mmcls.ImageClassifier',
    init_cfg=dict(type='Pretrained', checkpoint=teacher_ckpt),
```
Many users confuse the usage of `init_cfg`, and it is better to write directly in config

Reply:
It's a good point, but users may need to modify the config file themselves to run our code.

Reply:
It's just default ckpt, the user also can modify it by `cfg-options`

Reply:
Good point, fixed.